test/integration/fixtures/profile/profiles.json

```diff
@@ -522,7 +523,7 @@
           "signingAlgorithm": "JsonWebSignature2020",
           "signatureRepresentation": 0,
           "keyType": "ECDSASecp256k1DER",
-          "format": "jwt",
```

Comment:
Just to confirm - we are covering jwt use cases in other profiles/tests, correct ?

Reply:
yes